app/controllers/user_sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class UserSessionsController < ApplicationController
+  
+  skip_before_filter :require_login, except: [:destroy]
+
+  def new
+   @user = User.new
+  end
+
+  def create
+    if @user = login(params[:email], params[:password])
+      redirect_back_or_to(:users, notice: 'Login successful')
```

Comment:
we're gonna redirect to different pages depending on the user's role so please for now redirect to home page and simply display "login successful" message and also show logout link/button on home page if user is logged in